### PR TITLE
espeak builds on x86_64 now

### PR DESCRIPTION
**Pull request by dash102**

### CODE REVIEW

**Comment by dash102** [comment]
Fixing Issue https://github.com/haikuports/haikuports/issues/1041

File: app-accessibility/espeak/espeak-1.48.04.recipe [hunk 1]
```diff
@@ -39,18 +39,23 @@ BUILD_PREREQUIRES="
...
 {
 	cd src
-	make LIB_STDCPP=-lstdc++.r4 \
+	make LIB_STDCPP=$STDCPPLIBS \
 		AUDIO=wave LIB_PTHREAD= \
 		VISIBILITY_FLAG= \
 		PREFIX=$prefix \
 		DATADIR=$dataDir/espeak-data \
 		INCDIR=$includeDir/espeak
+
```
Comment (diversys) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: For this to work this block needs to be outside (above) BUILD() section.

File: app-accessibility/espeak/espeak-1.48.04.recipe [hunk 2]
```diff
@@ -35,22 +35,27 @@ BUILD_PREREQUIRES="
 	cmd:make
 	cmd:gcc
 	"
+AUDIO=wave LIB_PTHREAD= \
+VISIBILITY_FLAG= \
+PREFIX=$prefix \
+DATADIR=$dataDir/espeak-data \
+INCDIR=$includeDir/espeak
```
Comment (korli) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: These definitions don't belong here.

Comment (dash102) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: I think I misunderstood @diversys - what did you mean by your previous comment?

Comment (korli) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: Only the STDCPPLIBS change is required. You should also set STDCPPLIBS before using it.

Comment (dash102) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: Where should those definitions go? Back into build?

Comment (korli) [inline app-accessibility/espeak/espeak-1.48.04.recipe]: Yes

**Pull request merged**